lib/pengine/utils.c

```diff
@@ -992,7 +992,6 @@ unpack_operation(pe_action_t * action, xmlNode * xml_obj, pe_resource_t * contai
     int timeout_ms = 0;
     const char *value = NULL;
     const char *field = XML_LRM_ATTR_INTERVAL;
```

Comment:
I don't think there's any point in keeping this as a variable anymore

Reply:
Yeah TBH I'm not sure if there was a great reason for it to be there in the first place. I'll replace and remove.